lib/kdump_utils.pm

```diff
@@ -103,6 +103,11 @@ sub activate_kdump {
                 record_soft_failure 'default kdump memory size is too small, kdumptool gets killed by OOM, bsc#1120566';
             }
         }
+        # enable and verify fadump settings
+        if (get_var('FADUMP') && check_screen('yast2-fadump-not-enabled')) {
+            send_key 'alt-f';
```

Comment:
```suggestion
            send_key 'alt-f' if check_screen('yast2-fadump-not-enabled');
```

Reply:
like this we will always check that `yast2-fadump-enabled` in case variable is defined

Reply:
So we will keep as it is.